presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -928,10 +940,10 @@ public void testApproxPercentile()
 
         MaterializedResult actual = computeActual("" +
                 "SELECT orderstatus, " +
-                "   approx_percentile(orderkey, 0.5), " +
-                "   approx_percentile(totalprice, 0.5)," +
-                "   approx_percentile(orderkey, 2, 0.5)," +
-                "   approx_percentile(totalprice, 2, 0.5)\n" +
+                "   approx_percentile(orderkey, cast(0.5 as double)), " +
```

Comment:
Does inferring the type of the constant value work differently for Presto and Prestissimo ? The co-ordinator should infer the same type, then why are the casts being added here ?

Reply:
Reverted these casts, thanks for pointing it out.